performance_manager/lib/postgres_utils.py

```diff
@@ -25,24 +26,61 @@ def get_local_engine(
     process_logger = ProcessLogger("create_sql_engine")
     process_logger.log_start()
     try:
-        host = os.environ["DB_HOST"]
-        dbname = os.environ["DB_NAME"]
-        user = os.environ["DB_USER"]
-        port = int(os.environ["DB_PORT"])
-        password = urllib.parse.quote_plus(os.environ["DB_PASSWORD"])
+        db_host = os.environ.get("DB_HOST")
+        db_name = os.environ.get("DB_NAME")
+        db_password = os.environ.get("DB_PASSWORD", None)
+        db_port = os.environ.get("DB_PORT")
+        db_region = os.environ.get("DB_REGION", None)
+        db_user = os.environ.get("DB_USER")
+        db_ssl_options = ""
+
+        assert db_host is not None
+        assert db_name is not None
+        assert db_port is not None
+        assert db_user is not None
 
         process_logger.add_metadata(
-            host=host, database_name=dbname, user=user, port=port
+            host=db_host, database_name=db_name, user=db_user, port=db_port
```

Comment:
some slight renaming and using `get` here.

setting us up to get the AWS url if a password isn't provided (just like we do in the gtfs ingestion).